cgmanifest.json

```diff
@@ -60,7 +60,7 @@
 				"git": {
 					"name": "electron",
 					"repositoryUrl": "https://github.com/electron/electron",
-					"commitHash": "6f62f91822a80192cb711c604f1a8f1a176f328d"
+					"commitHash": "19c705ab80cd6fdccca3d65803ec2c4addb9540a"
```

Comment:
Sorry for the review on wrong line, the review ui is bad at commenting for unchanged line. 

Need to bump the version a few lines below `"version": "6.1.6"`

Reply:
Thanks, fixed in new commit